__tests__/Unit/Components/Tasks/TaskUpdateModal.test.tsx

```
@@ -1,26 +1,29 @@
-                </div>
-            )}
-        </div>
-    ))
+    jest.fn(({ isOpen, toggle, children }) =>
+        isOpen ? (
+            <div data-testid="mock-modal">
+                <button
+                    data-testid="close-modal"
+                    onClick={toggle}
+                    aria-label="Close"
+                >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)